DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -16,9 +16,10 @@
   <class name="pat::Lepton<reco::BaseTau>" />
 
   <!-- PAT Objects, and embedded data  -->
-  <class name="pat::Electron"  ClassVersion="29">
+  <class name="pat::Electron"  ClassVersion="30">
+   <version ClassVersion="30" checksum="3508125821"/>
    <version ClassVersion="29" checksum="1784986402"/>
-   <version ClassVersion="28" checksum="2518240031"/>
+   <version ClassVersion="28" checksum="1881133053"/>
```

Comment:
I don't think this is correct. 28 should correspond to the value used by ROOT6 which was added by Bill Tanenbaum in commit c5b318d75e06931185022968966be4179e0e00ae.